website/content/en/docs/golang/references/markers.md

```diff
@@ -203,6 +204,6 @@ customresourcedefinitions:
 
 [markers]:https://pkg.go.dev/sigs.k8s.io/controller-tools/pkg/markers
 [code-markers-design]:https://github.com/operator-framework/operator-sdk/blob/master/proposals/sdk-code-annotations.md
-[generate-csv-cli]:/docs/cli/operator-sdk_generate_csv
+[cli-gen-kustomize-manifests]:/docs/new-cli/operator-sdk_generate_kustomize_manifests
```

Comment:
Side note, what do we think about this: after we cut 0.19, should we remove all legacy docs and move `new-cli` to `cli`?

Reply:
@hasbro17 ^

Reply:
_Took me a while to find this comment since Github has it collapsed when it's marked as resolved. Un-resolving to make it visible for more discussion._

So I would think we want to remove the docs whenever we actually remove all the old CLI completely. 
Which if it's just going to be `v0.20.0`, then yeah I guess it would be right after `v0.19.0` is cut.